stylesheets/commons/Miscellaneous.less

```diff
@@ -2758,3 +2758,26 @@ Template: Sfrac
 	height: 1px;
 	overflow: hidden;
 }
+/*******************************************************************************
+Template: Weapon Skin box for Rainbow Six
+Author(s): Fenrir
+*******************************************************************************/
+.r6s-text-default {
+	color: var( --r6s-text-default, #a0a0a0 );
+}
+
+.r6s-text-uncommon {
+	color: var( --r6s-text-uncommon, #48c17b );
+}
+
+.r6s-text-rare {
+	color: var( --r6s-text-rare, #2d85bb );
+}
+
+.r6s-text-epic {
+	color: var( --r6s-text-epic, #8f32a6 );
+}
+
+.r6s-text-legendary {
+	color: var( --r6s-text-legendary, #e48d00 );
```

Comment:
i doubt those vars are defined

assumin they are not defined imo use some of the defined vars
mind that you need to decide which to use on dark and which on light mode

defined ("standard") color vars:
![Screenshot 2024-06-17 080312](https://github.com/Liquipedia/Lua-Modules/assets/75081997/66e7454d-e14f-4c14-9a4f-084711f18d83)
![Screenshot 2024-06-17 080348](https://github.com/Liquipedia/Lua-Modules/assets/75081997/5100d1f4-da5a-408e-a27b-38d21b850fea)
![Screenshot 2024-06-17 080420](https://github.com/Liquipedia/Lua-Modules/assets/75081997/6c79eaba-7757-49ba-a560-2301a40c7daf)

Reply:
The colors are exact copies of the colors from the game itself. Im not sure how to account for darkmode when the colors dont have an alternate, since they are from the game directly, im open to suggestions though. Mind even the Epic color (the darkest of the colors) isnt awful on darkmode
![image](https://github.com/Liquipedia/Lua-Modules/assets/58013431/1978a1be-33de-4e32-95c4-c513add4750f)
![image](https://github.com/Liquipedia/Lua-Modules/assets/58013431/6f3c9eaa-31f8-47f2-8d3b-45d96c0ef5a2)

Reply:
using undefined vars is a bad idea

also i wouldn't try to use the exact same colors as ingame
why not just use something close to it
especially for darkmode compatibility it would be good to use the already defined color vars

Reply:
Unless @Rathoz has a different opinion, when i asked the contributors for the wiki, they preferred the exact colors used in game at the risk they might be slightly hard to read on light or dark. In reality there are only 2 that might be difficult to see, the first one is sent above (the purple), the other ill provide here. The rest should have no issues in terms of visibility for light and dark
![image](https://github.com/Liquipedia/Lua-Modules/assets/58013431/a8c80a4d-04a3-4242-aa62-0b44fdd61961)
![image](https://github.com/Liquipedia/Lua-Modules/assets/58013431/6944d6b7-b408-4a70-b810-932bfb723e7e)

Reply:
using non existent css vars makes no sense still

and i think designers will want to have it dark mode compatible even if the color is slightly off from the ingame one

Reply:
> using non existent css vars makes no sense still

This we agree on. I just didnt do a good job of making these changes the first time around. Ill wait for Rathoz and the other employees to weigh in before moving forward with this

Reply:
What hjp said

Reply:
@liquidely

Reply:
You can remove the variable names since they are not defined. Instead, you can write it like this:
```
.r6s-text-default {
	color: #a0a0a0; // fallback for Bruinen

	.theme--light & {
		color: var( --clr-moon-10); //example with defined variable
	}

	.theme--dark & {
		color: #ffffff; // example with hex code
	}
}
```

As for the colors, Kevin told me they aim for at least WCAG AA accessibility. There are tools where you can test them, for example: https://webaim.org/resources/contrastchecker/
If you need help with which colors to use, let me know! We can set some colors with the help of Kevin.

Reply:
So, upon looking into this further, we can actually avoid this entire thing i think. The colors are based off of rarity for the item, these rarities also have icons (see the attached images). 
![image](https://github.com/user-attachments/assets/a2c3e81f-67f2-48e4-9fbd-1adc4d91d080)
![image](https://github.com/user-attachments/assets/2b5acc5c-d54b-49ae-a8b0-1390d87e08f4)
We could in theory just skip using the colors and instead use the icons to avoid causing issues